test/util/util.go

```diff
@@ -327,16 +327,28 @@ func ExpectCQResourceUsage(cq *kueue.ClusterQueue, flavor, resource string, v fl
 	}, Timeout, Interval).Should(gomega.Equal(v))
 }
 
-func SetAdmission(ctx context.Context, k8sClient client.Client, wl *kueue.Workload, admission *kueue.Admission) error {
+func SetQuotaReservation(ctx context.Context, k8sClient client.Client, wl *kueue.Workload, admission *kueue.Admission) error {
 	wl = wl.DeepCopy()
 	if admission == nil {
-		workload.UnsetAdmissionWithCondition(wl, "EvictedByTest", "Evicted By Test")
+		workload.UnsetQuotaReservationWithCondition(wl, "EvictedByTest", "Evicted By Test")
 	} else {
-		workload.SetAdmission(wl, admission)
+		workload.SetQuotaReservation(wl, admission)
 	}
 	return workload.ApplyAdmissionStatus(ctx, k8sClient, wl, false)
 }
 
+func SyncAdmissionFofWorkloads(ctx context.Context, k8sClient client.Client, wls ...*kueue.Workload) {
```

Comment:
add a comment for what this does and where it should be used (I suppose, in cases where the workload controller isn't running).

Maybe `SyncAdmittedConditionForWorkloads`